diff --git a/build.rs b/build.rs
index 17448c1..cbf6c2a 100644
--- a/alvr/server_openvr/build.rs
+++ b/alvr/server_openvr/build.rs
@@ -1,34 +1,16 @@
-use std::{env, path::PathBuf};
+use std::{env, path::{Path, PathBuf}};
 
 fn get_vulkan_headers_path(platform_name: &str) -> PathBuf {
-    alvr_filesystem::deps_dir()
-        .join(if platform_name == "linux" {
-            "linux"
-        } else {
-            "windows"
-        })
-        .join("vulkan-headers/include")
+    Path::new("@vulkan-headers@").to_owned()
 }
 
 fn get_ffmpeg_path(platform_name: &str) -> PathBuf {
-    let ffmpeg_path = alvr_filesystem::deps_dir()
-        .join(if platform_name == "linux" {
-            "linux"
-        } else {
-            "windows"
-        })
-        .join("ffmpeg");
-
-    if platform_name == "linux" {
-        ffmpeg_path.join("alvr_build")
-    } else {
-        ffmpeg_path
-    }
+    Path::new("@ffmpeg@").to_owned()
 }
 
 #[cfg(all(target_os = "linux", feature = "gpl"))]
 fn get_linux_x264_path() -> PathBuf {
-    alvr_filesystem::deps_dir().join("linux/x264/alvr_build")
+    Path::new("@x264@").to_owned()
 }
 
 fn main() {
